presentation/components/package.json

```diff
@@ -25,10 +26,11 @@
   },
   "scripts": {
     "compile": "npm run build",
-    "copy:locale": "cpx \"../common/assets/**/*\" ./lib/public && cpx \"./public/**/*\" ./lib/public",
-    "copy:assets": "cpx \"./src/**/*.{scss,json}\" ./lib",
-    "build": "npm run copy:assets && npm run copy:locale && npm run pseudolocalize && npm run extract && tsc 1>&2",
-    "build:watch": "npm run copy:assets && npm run copy:locale && npm run pseudolocalize && npm run extract && tsc -w",
+    "build": "npm run extract && npm run build:cjs && npm run build:esm",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs && npm run copy:cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm && npm run copy:esm",
+    "copy:cjs": "cpx \"../common/assets/**/*\" ./lib/cjs/public && cpx \"./public/**/*\" ./lib/cjs/public && cpx \"./src/**/*.{scss,json}\" ./lib/cjs",
```

Comment:
I see this was prior to your change, but I don't understand why we need to copy stuff from `common` into output of this package. I think that part should just be removed...

Reply:
done